src/signals/incident-management/containers/SignalingContainer/SignalingContainer.tsx

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import { Column, Heading, Row, themeSpacing } from '@amsterdam/asc-ui'
+import styled from 'styled-components'
+
+import { Signaling } from '../Dashboard/components'
+
+const StyledH1 = styled(Heading)`
+  margin-bottom: ${themeSpacing(8)};
+  margin-top: ${themeSpacing(6)};
+`
+export const SignalingContainer = () => {
+  return (
+    <Row>
+      <Column span={12}>
+        <StyledH1>Signalering</StyledH1>
+      </Column>
+      <Signaling />
+    </Row>
+  )
+}
```

Comment:
This seems a little odd. Why is the component moved into the Dashboard container/component but then a single SignallingContainer in the containers folder that imports the component from within Dashboard.

Maybe an idea to move the signalling component back to `src/signals/incident-management/components/` and both Dashboard and Signalling container can refer to that.

Reply:
well actually, this is the implementation needed for signalling without dashboard. It's like an augmentation. If we're eventually moving signaling to dashboard, why not place it there.

Reply:
I get why there is a Signalling container. But not why the actual component is inside the Dashboard container while it is shared among two different containers. Also it is not sure if we will eventually move signalling into the dashboard. It is a choice for every municipality if they want to have the dashboard feature or not.